acorn-loose/src/statement.js

```diff
@@ -282,47 +282,52 @@ lp.parseClass = function(isStatement) {
   if (this.curIndent + 1 < indent) { indent = this.curIndent; line = this.curLineStart }
   while (!this.closes(tt.braceR, indent, line)) {
     if (this.semicolon()) continue
-    let method = this.startNode(), isGenerator, isAsync
+    let element = this.startNode(), isGenerator, isAsync, kind = "method"
     if (this.options.ecmaVersion >= 6) {
-      method.static = false
+      element.static = false
       isGenerator = this.eat(tt.star)
     }
-    this.parsePropertyName(method)
-    if (isDummy(method.key)) { if (isDummy(this.parseMaybeAssign())) this.next(); this.eat(tt.comma); continue }
-    if (method.key.type === "Identifier" && !method.computed && method.key.name === "static" &&
-        (this.tok.type !== tt.parenL && this.tok.type !== tt.braceL)) {
-      method.static = true
+    this.parseClassElementName(element)
+    if (isDummy(element.key)) { if (isDummy(this.parseMaybeAssign())) this.next(); this.eat(tt.comma); continue }
+    if (element.key.type === "Identifier" && !element.computed && element.key.name === "static" &&
+        this.tok.type !== tt.parenL && this.tok.type !== tt.braceL && this.tok.type !== tt.eq &&
+        this.tok.type !== tt.semi) {
+      element.static = true
       isGenerator = this.eat(tt.star)
-      this.parsePropertyName(method)
+      this.parseClassElementName(element)
     } else {
-      method.static = false
+      element.static = false
     }
-    if (!method.computed &&
-        method.key.type === "Identifier" && method.key.name === "async" && this.tok.type !== tt.parenL &&
-        !this.canInsertSemicolon()) {
+    if (!element.computed &&
+        element.key.type === "Identifier" && element.key.name === "async" && this.tok.type !== tt.parenL &&
+        this.tok.type !== tt.eq && this.tok.type !== tt.semi && !this.canInsertSemicolon()) {
       isAsync = true
       isGenerator = this.options.ecmaVersion >= 9 && this.eat(tt.star)
-      this.parsePropertyName(method)
+      this.parseClassElementName(element)
     } else {
       isAsync = false
     }
-    if (this.options.ecmaVersion >= 5 && method.key.type === "Identifier" &&
-        !method.computed && (method.key.name === "get" || method.key.name === "set") &&
-        this.tok.type !== tt.parenL && this.tok.type !== tt.braceL) {
-      method.kind = method.key.name
-      this.parsePropertyName(method)
-      method.value = this.parseMethod(false)
+    if (this.options.ecmaVersion >= 5 && element.key.type === "Identifier" &&
+        !element.computed && (element.key.name === "get" || element.key.name === "set") &&
+        this.tok.type !== tt.parenL && this.tok.type !== tt.braceL && this.tok.type !== tt.eq &&
+        this.tok.type !== tt.semi) {
```

Comment:
`get`/`set` is a property if it precedes a line break:

```js
test(`class C {
  set
  *aaa(){}
}`, {}, {ecmaVersion: 13})
```

Reply:
Fixed in https://github.com/acornjs/acorn/pull/1026/commits/464858c92dad71887aed46352cdc2381680bf324.

Also fixed in https://github.com/acornjs/acorn/pull/1026/commits/24aee3780ec119417d4251b69da559de8a6e1aef about `new.target`.